docs/google/best_practices.md

```diff
@@ -129,9 +129,18 @@ One word of caution is there is a limit to the total number of repetitions.  Tak
 that your parameter sweeps, especially products of sweeps, do not become so excessively large
 that they overcome this limit.
 
+## Use batches if sweeps are not possible
+
+The engine has a method called `run_batch()` that can be used to send multiple
+circuits in a single request.  This can be used to increase the efficiency
+and rep rate of your program.
```

Comment:
repetition rate? Maybe we want to define that concept too?

Reply:
Done.